app/views/users/_preferences.html.erb

```diff
@@ -30,8 +30,10 @@ See COPYRIGHT and LICENSE files for more details.
   <%= render Settings::TimeZoneSettingComponent.new(
     "time_zone",
     form: pref_fields,
+    include_blank: false,
     container_class: (defined? input_size) ? "-#{input_size}" : "-wide"
   )
+
```

Comment:
Unneeded new line

Reply:
Removed in [b2e6d67](https://github.com/opf/openproject/pull/16567/commits/b2e6d67d19a5355cc2caed4fef5621330b3bbc38)